data_steward/cdr_cleaner/cleaning_rules/deid/generalize_cope_insurance_answers.py

```diff
@@ -81,7 +90,10 @@ def __init__(self,
                          table_namer=table_namer)
 
     def get_sandbox_tablenames(self):
```

Comment:
For context, can you elaborate on why this function is defined, but appears never to be used in the project?

Reply:
`get_sandbox_tablenames()` was introduced in [DC-734](https://precisionmedicineinitiative.atlassian.net/browse/DC-734) as part of the base cleaning rule created in [DC-712](https://precisionmedicineinitiative.atlassian.net/browse/DC-712). The intention was to have a standardized way of listing the table names in the sandbox. However, in most of the cleaning rules, there are only one or zero sandbox tables. That caused this function to be not utilized in some cleaning rules (however it is defined in each of the cleaning rules because the base class has `get_sandbox_tablenames()` as the abstract method from DC-734). Also, there were already several cleaning rules before the base class was created by DC-712. For some of those rules, `get_sandbox_tablenames()` was added but just left to raise `NotImplementedError`.